themes/default/content/blog/iac-recommended-practices-structuring-pulumi-projects/index.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "Iac Recommended Practices: Structuring Pulumi Projects"
+date: 2023-03-16T16:50:38-07:00
+meta_desc: The third post in the Zephyr series examines recommended practices around structuring Pulumi projects.
+meta_image: meta.png
+authors:
+    - scott-lowe
+    - christian-nunciato
+    - aaron-kao
+tags:
+    - best-practices
+    - cloud-engineering
+    - zephyr
+---
+
+This is the third post in a series of blog posts focused on Zephyr Archaeotech Emporium---our fictional company---and their use of Pulumi to manage their online retail store. In the first post, you saw how Zephyr initially decided to go with a single Pulumi project for managing deployments of their online retail store application. In this post, you'll see how Zephyr's use of Pulumi evolves as their company grows and evolves.<!--more-->
+
+The [first blog post](/blog/iac-recommended-practices-code-organization-and-stacks/) in the Zephyr series specifically mentions that the series will show "how Zephyr's use of Pulumi evolves as the company grows and their online retail store application changes to accommodate their growth." While the ultimate goal of this series is to discuss recommended practices for using Pulumi to manage a fairly complex containerized application, it's important to note that these recommended practices can and will change over time as the organizations using Pulumi also change and evolve.
+
+Here are links to all the blog posts in the series (entries below that are not linked are planned but haven't yet been published---this will get updated as new posts are published):
+
+[IaC Recommended Practices: Code Organization and Stacks](/blog/iac-recommended-practices-code-organization-and-stacks/)
+
+[IaC Recommended Practices: Developer Stacks and Git Branches](/blog/iac-recommended-practices-developer-stacks-git-branches/)
+
+**IaC Recommended Practices: Structuring Pulumi Projects** (this post)
+
+IaC Recommended Practices: Local Testing with Pulumi
+
+IaC Recommended Practices: Tying Stacks Together with Stack References
+
+IaC Recommended Practices: Evolving the Application
+
+IaC Recommended Practices: Adding Pulumi Deployments
+
+IaC Recommended Practices: Refactoring for Reuse
+
+## Catching up with Zephyr
+
+As a result of launching its online presence, Zephyr saw amazing success. The market for arcane artifacts and novel curiosities is booming, and Zephyr is becoming a leader in this market. Funded by this success, Zephyr has grown rapidly. Along with this growth, Zephyr's software engineering and IT teams also grew and expanded. Fortunately, Zephyr's [use of per-developer stacks with short-lived ephemeral Git branches](/blog/iac-recommended-practices-developer-stacks-git-branches/) positioned the development teams well for this growth.
+
```

Comment:
I like how the 2nd post drops this as a note to the first post. we can do the same pointing to the 2nd post's code

{{% notes %}}
For a snapshot of the code as it was at the end of the first post in the series, see the [`original`](https://github.com/pulumi/zephyr-app/tree/original) tag of the [`pulumi/zepyhr-app`](https://github.com/pulumi/zephyr-app) repository on GitHub.
{{% /notes %}}

Reply:
Good idea, I'll update the post. We should just make this part of every post's structure moving forward.

Reply:
Addressed in e569521b